src/QA/QA.Web/Views/Review/Review.cshtml

```diff
@@ -267,22 +239,38 @@
             </div>
         </div>
     </div>
-
     <div class="grid-row">
         <div class="column-full">
             <hr class="no-bottom-margin">
         </div>
     </div>
-
+    <div class="grid-row">
+        <div class="column-full">
+            <h2 class="heading-large">Application process</h2>
+        </div>
+    </div>
+    <div class="grid-row">
+        <div class="column-full">
+            <p asp-condition="@Model.HasSpecifiedThroughFaaApplicationMethod">Candidates will apply through the find an apprenticeship service</p>
+            <div asp-condition="@Model.HasApplicationUrl">
```

Comment:
Should we add HasSpecifiedThroughExternalApplicationMethod to QA too rather than this assumption?